setup.sh

```diff
@@ -448,6 +449,26 @@ function install_home()
   update_version
 }
 
+function show_current_env()
+{
+  local config_files_path="$etcdir"
+  local config_file_template="$config_files_path/strings/current_env_content.txt"
+  local shellrc="${HOME}/.bashrc"
+  local content_current_env_file
+
+  if [[ -f "$config_file_template" ]]; then
+    content_current_env_file="$(cat "${config_file_template}")"
+    if ! grep -Fxq "$content_current_env_file" "$shellrc"; then
+      safe_append "$content_current_env_file" "$shellrc"
+      warning ' -> For a better experience with kw, please, open a new terminal.'
+    fi
+  else
+    warning "Setup could not find ${config_file_template}"
+    warning 'Try running ./setup -i'
+    return 2
```

Comment:
Add the 2 error comment.

Reply:
I didn't understand very well. Would it be writing a comment explaining why to run `./setup -i`?